### ISSUE TITLE
defrost windows

### ISSUE THREAD

**Issue by ndit-dev** [irritanterik/homey-tesla.com#33]
Cannot find a way to turn on the window defrosting from flows. Is this not possible through the API, or is it just not implemented here?

**Comment by irritanterik** [comment]
Is this possible with the native Tesla app? I don't see a specific command for defrost in the (unofficial) api documentation (https://www.teslaapi.io/vehicles/commands#start-hvac-system). It is possible to set the temperature and activate climate control (airco) with flow cards.

**Comment by ndit-dev** [comment]
Yes, possible from the native app
![CA8EE5F0-DB5E-4FA5-BC19-55A61CA025F1](https://user-images.githubusercontent.com/8581030/68918652-5eb66f00-076f-11ea-9b2e-8566e10e2e1d.jpeg)

**Comment by Genghes** [comment]
My thoughts:
The API is not official or open, it’s reverse engineered and it will probably take some time to find this Relatively new feature.

**Comment by irritanterik** [comment]
Agreed. I'm watching the unofficial api documentation repo for changes on this matter.

**Comment by robkor** [comment]
Defrost is the same as preconditioning to Max temperature. I found this API-command:
https://tesla-api.timdorr.com/vehicle/commands/climate#post-api-1-vehicles-id-command-set_preconditioning_max
